samcli/local/docker/container.py

```diff
@@ -55,6 +55,7 @@ def __init__(
         docker_client=None,
         container_opts=None,
         additional_volumes=None,
+        container_host=None,
```

Comment:
same here, this is not optional

Reply:
`_init_` here is also used in another its child class `LambdaBuildContainer`. The new option is not used in that class so I make it as optional.

Reply:
if this is used by other classes (i'm assuming those other classes are instantiated outside of sam local suite, is it sam build?) , should we set a default value? this can get overridden when the actual option is used.

Reply:
As far as I know, `container_host` is only used in SAM LOCAL suite when one of its child class is instantiated. So whether or not setting the default value here makes no difference. If we could potentially expand the use of new option outside of sam local suite, then yes we should set default to `localhost` here. Considering there is no negative impact setting the default value, I will add the default value to the option here.